lib/mix/lib/mix/release.ex

```diff
@@ -786,9 +790,11 @@ defmodule Mix.Release do
   The exact chunks that are kept are not documented and may change in
   future versions.
   """
-  @spec strip_beam(binary()) :: {:ok, binary} | {:error, :beam_lib, term}
-  def strip_beam(binary) do
-    case :beam_lib.chunks(binary, @significant_chunks, [:allow_missing_chunks]) do
+  @spec strip_beam(binary(), List.t()) :: {:ok, binary} | {:error, :beam_lib, term}
+  def strip_beam(binary, options \\ [keep: []]) when is_list(options) do
+    case :beam_lib.chunks(binary, Enum.uniq(@significant_chunks ++ options[:keep]), [
```

Comment:
I would default to `[]` (that's a common default value when passing `options`) and get the info from `options` assuming the `:keep` key might not be there. In short words, this should work `strip_beam(beam, [])` 🙂 


Ah! And I would also change the `List.t()` in the spec to `keyword()` here and in `parse_strip_beams_option`

Reply:
I thought you meant the opposite from your first comment, when you said that calling strip_beam like this felt strange: `Mix.Release.strip_beam(beam, ['Docs', 'Dbgi'])` :)

Reply:
Oh! the `[]` in `strip_beam(beam, [])` is the `options` not the list of chunks to keep. I think some code would explain it better:

```suggestion
  @spec strip_beam(binary(), keyword) :: {:ok, binary} | {:error, :beam_lib, term}
  def strip_beam(binary, options \\ []) when is_list(options) do
    chunks_to_keep = Keyword.get(options, :keep, [])
    case :beam_lib.chunks(binary, Enum.uniq(@significant_chunks ++ chunks_to_keep), [
```

Sorry for the misunderstanding 😅

Reply:
Oh, you mean the list still having the keep inside, but omitting it from the signature :)

Reply:
LOL i wrote the exact same code you posted :)